.github/workflows/Test-AITs.yml

```
@@ -109,6 +110,14 @@ jobs:
       fail-fast: false
       matrix: ${{ fromJson(needs.list-tests.outputs.tests) }}
     steps:
+      - name: Retrieve agent from cache
+        id: retrieve-agent
+        uses: actions/cache@v3
+        with:
+          path: /home/runner/work/newrelic-java-agent/newrelic-java-agent/newrelic-agent/build/newrelicJar/newrelic.jar
+          key: ${{ github.run_id }}
+          fail-on-cache-miss: true
+
```

Fail fast if cache misbehaves.